product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: "Preparing your Azure account"
+
+---
+
+BigAnimal requires you to configure your Azure subscription, before you deploy your clusters. The configurations that you perform ensure that your Azure subscription is prepared to meet your clusters' resource requirements, such as: 
+
+ * Is there a sufficient limit on the number vCPU or Public IP addresses in your region?
+ * Is there a restriction on VM size SKUs for the Standard Dsv2 and Standard Esv3 VM size families?
+ * Are the necessary Azure Service Providers registered for your subscription?
+
+Before proceeding, see [Understanding requirements in Azure](../01_preparing_azure/01_understanding_qotas_in_azure) for information about planning for your clusters' resource requirements in Azure.
+
+## Step 1: Check the Azure requirements
+
+Although EDB recommends using the `azure-preflight` script, you also have the option of using the Azure CLI or the Azure Portal to manually check the requirements.
+
+ * [Method 1: Use the Azure shell script](#use-the-azure-shell-script) (recommended)
+ * [Method 2: Manually check requirements](#manually-check-requirements)
+
+
+### Use the Azure shell script
+
+EDB provides a shell script called [`azure-preflight`](https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/azure-preflight.sh) that automatically checks requirements in your Azure subscription based on the clusters you plan to deploy.
+
+1. Open the [Azure Cloud Shell](https://shell.azure.com/) in your browser.
+ 
+ !!! Note
+     Before proceeding, ensure that the [jq command-line JSON processor](https://stedolan.github.io/jq/) utility is installed in your environment.
+
+1. From the Azure Cloud Shell, run the following command to download the script:  
+
+ ```
+ curl -fsSL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/azure-preflight.sh
+ ```
+
+1. Run the `azure-preflight.sh` script with the following options: 
+   - *azure_region* is the Azure region you plan to deploy your clusters.
+   - *pg_type* is virtual machine instance of type E{N}Sv3 or D{N}Sv2 using {N} cores.
+   - *end_point* indicates either `public` or `private` endpoint.
+   - *--ha* indicates that the instance is run with high availability enabled.
+   - *--with-infra* indicates that no clusters have been created in the region.
+
+  
+ ```
+  ./cloud-utilities/azure/azure-preflight.sh --location <azure_region> --pgtype <pg_type> --endpoint <end_point> [--ha] [--with-infra]
```

Comment:
This invocation probably isn't correct - once the `curl` above is fixed, the script won't live at `./cloud-utilities`

Reply:
Will change this as per the discussion in the above comment.